kube/kong/kong-db-service.yaml

```diff
@@ -0,0 +1,14 @@
+# Service to expose the kong-db to the kong pod
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: kong-db
+  name: kong-db
+spec:
```

Comment:
Might be worth explicitly adding that this is `ClusterIP` in here - although it's the default, makes it easier to understand what's going on?